packages/gatsby/package.json

```diff
@@ -88,6 +88,7 @@
     "lodash": "^4.17.10",
     "md5": "^2.2.1",
     "md5-file": "^3.1.1",
+    "micromatch": "^3.1.10",
```

Comment:
Need to update the top level `yarn.lock` as well

Reply:
Should be fine, workspaces aren't currently part of the lockfile (and `micromatch@^3.1.10` already was part of it).